detekt_custom.yml

```
@@ -1258,6 +1258,7 @@ datadog:
       # region Opentelemetry
       - "io.opentelemetry.api.trace.TracerBuilder.build()"
       - "io.opentelemetry.api.trace.TracerBuilder.setInstrumentationVersion(kotlin.String?)"
+      - "io.opentelemetry.api.trace.SpanBuilder.setAttribute(kotlin.String?, kotlin.String?)"
```

minor: just ordering

```suggestion
      # region OpenTelemetry
      - "io.opentelemetry.api.trace.SpanBuilder.setAttribute(kotlin.String?, kotlin.String?)"
      - "io.opentelemetry.api.trace.TracerBuilder.build()"
      - "io.opentelemetry.api.trace.TracerBuilder.setInstrumentationVersion(kotlin.String?)"
```